### ISSUE TITLE
Handle Gitlab subgroups

### ISSUE THREAD

**Issue by mspangdal** [tomasbjerre/git-changelog-lib#74]
The {{ownerName}} and {{repoName}} doesn't seem to be correctly populated given a gitlab url such as:
https://gitlab.server/Group/SubGroup/Project.git

In this case ownerName is set to SubGroup and Group is ignored
More on subgroups in Gitlab: https://docs.gitlab.com/ee/user/group/subgroups/

For our use case it would be sufficient if Subgroups are treated as part of the Project name  ownerName is set to the parent Group as would be expected

But I suppose this introduces more edge cases where gitlab is not hosted at the root of the domain?

**Comment by tomasbjerre** [comment]
Are you working on a PR for this @mspangdal ?

How do you use the library? Jenkins plugin?

**Comment by mspangdal** [comment]
Yeah, Jenkins plugin.  I don't have a clear idea of how to solve it yet so just looking at the code for now

**Comment by mspangdal** [comment]
Thats a good point,  quick way forward!
